helm/cloud-provider-vsphere/values.yaml

```diff
@@ -55,3 +55,8 @@ vsphere-csi-driver:
     runAsNonRoot: false
     seccompProfile:
       type: RuntimeDefault
+
+  initContainer:
+    image:
+      name: giantswarm/kubectl
+      tag: 1.29.9
```

Comment:
Would it make sense to add an Renovate annotation here?

Reply:
yes, thank you for the reminder - i had forgotten to do this :+1:

Reply:
actually i remember why i didn't add this - we don't want to be using the latest kubectl because of version skew. the CSI has a compatibility matrix and we want to ensure that the kubectl version used here stays within the versions of kubernetes which the vsphere CSI release supports.

Reply:
Ah, ok. Then we should at least find a way to keep it aligned and inside the support matrix.

Reply:
yeah i plan on adding some docs at least, but ideally the makefile would handle it